torch/fx/graph_module.py

```diff
@@ -194,10 +194,19 @@ def __reduce__(self):
     def __deepcopy__(self, memo):
         fake_mod = torch.nn.Module()
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
-        return GraphModule(fake_mod, self.graph)
+        # delete generated attributes before creating a new
+        # GraphModule
+        generated_attrs = ['code', '_graph', '_modules']
+        for attr in generated_attrs:
+            fake_mod.__dict__.pop(attr, None)
+        graph_module = GraphModule(fake_mod, self.graph)
+        graph_module.__dict__.update(fake_mod.__dict__)
+        return graph_module
 
     def __copy__(self):
-        return GraphModule(self, self.graph)
+        graph_module = GraphModule(self, self.graph)
+        graph_module.__dict__ = self.__dict__
```

Comment:
Do we need to delete the generated attributes here as well?

Reply:
I don't think we can delete them, since it is used by the old module, but I skipped overwriting them